bot/exts/info/tags.py

```diff
@@ -354,16 +274,45 @@ def tag_sort_key(tag_item: tuple[TagIdentifier, Tag]) -> str:
 
         return result_lines
 
-    def accessible_tags_in_group(self, group: str, user: discord.Member) -> list[str]:
+    def accessible_tags_in_group(self, group: str, user: Member | User) -> list[str]:
         """Return a formatted list of tags in `group`, that are accessible by `user`."""
         return sorted(
             f"**\N{RIGHT-POINTING DOUBLE ANGLE QUOTATION MARK}** {identifier}"
             for identifier, tag in self.tags.items()
             if identifier.group == group and tag.accessible_by(user)
         )
 
-    @tags_group.command(name="get", aliases=("show", "g"), usage="[tag_group] [tag_name]")
-    async def get_command(self, ctx: Context, *, argument_string: Optional[str]) -> bool:
+    async def get_command_ctx(
+        self,
+        ctx: Context,
+        name: str
+    ) -> bool:
+        """Made specifically for `error_handler.py`, See `get_command` for more info."""
```

Comment:
Docstrings should explain what the method does in a nutshell, you can also elaborate as much as you want when it comes to **why** it was created & **where** it will be needed.

Reply:
I mean it would be just copying the docstring of `get_command` and also I have seen these types of docstring referring to other method for more info in the codebase, some examples are in `helpers.py`. So I think that answers the **why** and I have already included the **where** it will be used.

So shall I still change it?